### PR TITLE
Fixing a couple of keytool-related doc bugs.

### PR DESCRIPTION
**Pull request by sfshaza2**
Fixes https://github.com/flutter/website/issues/3795
Fixes https://github.com/flutter/website/issues/2066

### CODE REVIEW

File: src/docs/deployment/android.md [hunk 1]
```diff
@@ -74,10 +74,11 @@ On Windows, use the following command:
...
 
-{{site.alert.warning}}
-  Keep the `keystore` file private;
-  do not check it into public source control.
-{{site.alert.end}}
+This command stores the `key.jks` file in your home
+directory. If you want to store it elsewhere, change
+the argument you pass to the `-keystore` parameter.
+**However, keep the `keystore` file private;
+do not check it into public source control!**
```
Comment (kwalrath) [inline src/docs/deployment/android.md:81]: do not -> don't

(per Google style)

File: src/docs/deployment/android.md [hunk 2]
```diff
@@ -87,8 +88,9 @@ keytool -genkey -v -keystore c:/Users/USER_NAME/key.jks -storetype JKS -keyalg R
   'Java binary at:'. Then use that fully qualified path
   replacing `java` (at the end) with `keytool`.
   If your path includes space-separated names,
-  such as `Program Files`, place quotes around the
-  space-separated names. For example: `/"Program Files"/`
+  such as `Program Files`, use a delimeter appropriate
```
Comment (kwalrath) [inline src/docs/deployment/android.md:91]: delimeter -> delimiter... but I'm not sure _delimiter_ is the word we want here. Looking up the definition of delimiter, it's a character that separates parts of a string. Maybe:

...use platform-appropriate notation for the names. For example, on Mac/Linux use ..., and on Windows use ....

File: src/docs/deployment/android.md [hunk 3]
```diff
@@ -88,9 +88,10 @@ do not check it into public source control!**
   'Java binary at:'. Then use that fully qualified path
   replacing `java` (at the end) with `keytool`.
   If your path includes space-separated names,
-  such as `Program Files`, use a delimeter appropriate
-  to your platform. For example, use `/"Program Files"/`
-  on Mac/Linux, or `/Program\ Files/` on Windows.
+  such as `Program Files`, use platform-appropriate
+  notation for the names. For example, on Mac/Linux
+  use `/"Program Files"/`, and on Windows use
```
Comment (kwalrath) [inline src/docs/deployment/android.md:93]: Text looks good, but I just noticed that this doesn't render correctly (at least in https://flutter.dev/docs/deployment/android). Should it be these instead?

`"Program Files"`
`Program\ Files`

(And actually, I think the first one might work for both. Maybe we should get a DPE to advise?)

**Comment by sfshaza2** [comment]
I saw your note. I will play with it.  :)

File: src/docs/deployment/android.md (hunk 3 above)
Comment (sfshaza2) [reply src/docs/deployment/android.md:93]: @kwalrath, I've staged the site and it looks fine to me. Please confirm:

https://sz-flutter-2.firebaseapp.com/docs/deployment/android

**Comment by sfshaza2** [comment]
@kwalrath, I think the formatting is ok.  I've included a staging link this time. Please confirm.

File: src/docs/deployment/android.md (hunk 3 above)
Comment (kwalrath) [reply src/docs/deployment/android.md:93]: The formatting does look better than before (not cut off any more), but are the `\` characters really supposed to be in the strings? And why would there be a `\` _after_ the `"`? This is just a command-line string, not a regex.

image

**Comment by sfshaza2** [comment]
@johnpryan, can you weigh in? The staged version seems good to me.

File: src/docs/deployment/android.md (hunk 3 above)
Comment (johnpryan) [reply src/docs/deployment/android.md:93]: I think shells on macOS and Linux support escaping using the space character (`cd Program\ Files/`) and the Windows command prompt needs the path is wrapped quotation marks (`cd "C:\Program Files"`)

File: src/docs/deployment/android.md [hunk 4]
```diff
@@ -438,21 +444,20 @@ The resulting app bundle or APK files are located in
...
 [appid]: {{site.android-dev}}/studio/build/application-id
 [applicationtag]: {{site.android-dev}}/guide/topics/manifest/application-element
 [arm64-v8a]: {{site.android-dev}}/ndk/guides/abis#arm64-v8a
 [armeabi-v7a]: {{site.android-dev}}/ndk/guides/abis#v7a
-[crash-issue]: https://issuetracker.google.com/issues/147096055
-[x86-64]: {{site.android-dev}}/ndk/guides/abis#86-64
 [bundle]: {{site.android-dev}}/platform/technology/app-bundle
 [bundle2]: {{site.android-dev}}/guide/app-bundle
 [configuration qualifiers]: {{site.android-dev}}/guide/topics/resources/providing-resources#AlternativeResources
+[crash-issue]: https://issuetracker.google.com/issues/147096055
```
Comment (johnpryan) [inline src/docs/deployment/android.md:457]: Is this link publicly accessible?

File: src/docs/deployment/android.md (hunk 3 above)
Comment (kwalrath) [reply src/docs/deployment/android.md:93]: The `\` characters shouldn't be in the examples, right?

Comment (kwalrath) [reply src/docs/deployment/android.md:93]: Also, iirc quotation marks work for Mac/Linux as well as Windows. (Though we might or might not want to say that.)

File: src/docs/deployment/android.md (hunk 4 above)
Comment (kwalrath) [reply src/docs/deployment/android.md:457]: You have to sign in, but it accepted my gmail account... so probably?

Comment (johnpryan) [reply src/docs/deployment/android.md:457]: 👍

File: src/docs/deployment/android.md (hunk 3 above)
Comment (johnpryan) [reply src/docs/deployment/android.md:93]: Yeah I would change this to "on Mac/Linux use `Program\ Files` and on Windows use `"Program Files"`

**Pull request merged**